test/MfldSetTac.lean

```diff
@@ -24,7 +24,7 @@ section stub_lemmas
 structure PartialHomeomorph (α : Type u) (β : Type u) extends PartialEquiv α β
 
 noncomputable
-instance PartialHomeomorph.has_coe_to_fun : CoeFun (PartialHomeomorph α β) (λ _ => α → β) := test_sorry
+instance PartialHomeomorph.has_coe_to_fun : CoeFun (PartialHomeomorph α β) (fun _ ↦  α → β) := test_sorry
```

Comment:
Here, below and in `test/toAdditive` you have a whitespace too much (maybe somewhere else as well?)

Reply:
Thanks; fixed (here and in all of mathlib). This means this PR has some drive-by space changes.